snap/snapcraft.yaml

```diff
@@ -14,6 +14,12 @@ layout:
   /usr/share/locale:
     bind: $SNAP/usr/share/locale
 
+plugs:
+  read-snap-data:
```

Comment:
I think the convention required for naming the plug says this should map the path exposed, which in this case would simply be "snap", but that does feel weird to me.  Can you please s/read-snap-data/read-snap/ and we'll see what they say?

Reply:
Done.